NewForce-Cohort-7/tabloidmvc-mollasses-gang
# Ticket 24 ability to view user details

[Description](https://trello.com/c/Ogoi3dYK/25-24-view-user-profile-details)
Added the feature that when an admin is viewing the list of users they can click on details and will be able to view all listed details of the user. Given that a user is logged in and selects the Users they should be directed to a list view of all the current users. To the right of each user there will be links for [Edit, Details, Delete}, clicking on the details will bring you to a view of all a users details.  (https://trello.com/c/Ogoi3dYK/25-24-view-user-profile-details)

Testing Instructions

git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout mc1
If you don't see the most recent version of my files, git pull origin
If you haven't created the Tabloid MVC database, please do so.
Use this to fill out tables and columns: [SQL data] (https://github.com/NewForce-Cohort-7/tabloidmvc-mollasses-
gang/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
Use this to seed the database: [SQL Seed Data] (https://github.com/NewForce-Cohort-7/tabloidmvc-mollasses-gang/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
Click the execute solid green button to create database


SELECT * FROM UserProfile  it will display all current users in the database and their details.
Click the execute solid green button to view the database
You will see Id |DisplayName |FirstName| LastName| DisplayName | Email |CreateDateTime| ImageLocation| UserTypeId





Click the execute with debugger solid green button to launch the app

In browser, please go to Login on the nav bar and log in with this email: [admin@example.com](mailto:admin@example.com)
This should redirect you back to the home page.
User should be showing on the nav bar now. Please click this to view a list of users.
You will see 3 items | FullName | DisplayName | UserTypeId
To the right of full name you will see links for (Edit, Details, Delete)
From this view you will click on "DETAILS" and that will bring you to a view of that users details. 
You will see a list with 9 items
 Id 
 FirstName 
 LastName 
 DisplayName 
 Email |
CreateDateTime 
ImageLocation 
 UserTypeId
 FullName
To the right of full name you will see links for (Edit, Details, Delete)

From this view you will click on "DETAILS" and that will bring you to a view of that users details. 
Log out should be showing on the nav bar now. Please click this to log out.


Checklist:
My code follows the style guidelines of this project
I have performed a self-review of my code
I have commented my code, particularly in hard-to-understand areas
My changes generate no new warnings or errors
I have added test instructions that prove my fix is effective or that my feature works
